test/PowerShellEditorServices.Test/Session/ScriptFileTests.cs

```diff
@@ -595,9 +595,14 @@ public void DocumentUriRetunsCorrectStringForAbsolutePath()
                 scriptFile = new ScriptFile(path, path, emptyStringReader, PowerShellVersion);
                 Assert.Equal("file:///c%3A/Users/AmosBurton/projects/Rocinate/ProtoMolecule.ps1", scriptFile.DocumentUri);
 
-                path = @"c:\Users\BobbyDraper\projects\Rocinate\foo's_~#-[@] +,;=%.ps1";
+                path = @"c:\Users\BobbieDraper\projects\Rocinate\foo's_~#-[@] +,;=%.ps1";
                 scriptFile = new ScriptFile(path, path, emptyStringReader, PowerShellVersion);
-                Assert.Equal("file:///c%3A/Users/BobbyDraper/projects/Rocinate/foo%27s_~%23-%5B%40%5D%20%2B%2C%3B%3D%25.ps1", scriptFile.DocumentUri);
+                Assert.Equal("file:///c%3A/Users/BobbieDraper/projects/Rocinate/foo%27s_~%23-%5B%40%5D%20%2B%2C%3B%3D%25.ps1", scriptFile.DocumentUri);
+
+                // Test UNC path
+                path = @"\\ClarissaMao\projects\Rocinate\foo's_~#-[@] +,;=%.ps1";
```

Comment:
Do we know how a network drive path resolves on *nix? Is it "smb://server/place"?

Reply:
I don't know if you can actually open a network drive path in vscode 🤔

Reply:
Wouldn't `smb:` be its own protocol then, different from `file:`?  If that is indeed the case, we could add another condition at the beginning when we return early for `file:` and `untitled:` i.e. return early for `smb:` as well.

Reply:
We discussed this and decided this is a Windows-specific issues, since other OSes transparently mount network shares to the filesystem, meaning we don't need to deal with it